glide-core/src/errors.rs

```diff
@@ -0,0 +1,29 @@
+use redis::RedisError;
+
+pub enum RequestErrorType {
+    Unspecified,
```

Comment:
```suggestion
    Unspecified = 0,
```
to ensure 1:1 mapping for FFI clients